miqa/core/rest/accounts.py

```diff
@@ -82,3 +86,18 @@ def post(self, request):
         if request.user.is_authenticated:
             logout(request)
         return Response(status=status.HTTP_204_NO_CONTENT)
+
+
+class AccountSignupForm(SignupForm):
+    def __init__(self, *args, **kwargs):
+        super(AccountSignupForm, self).__init__(*args, **kwargs)
+        self.fields['first_name'] = forms.CharField(
+            label=('First Name'),
+            min_length=1,
+            widget=forms.TextInput(attrs={'placeholder': ('First name')}),
+        )
+        self.fields['last_name'] = forms.CharField(
+            label=('Last Name'),
+            min_length=1,
+            widget=forms.TextInput(attrs={'placeholder': ('Last name')}),
+        )
```

Comment:
```suggestion
    first_name = forms.CharField(
        label=('First Name'),
        min_length=1,
        widget=forms.TextInput(attrs={'placeholder': ('First name')}),
    )
    last_name = forms.CharField(
        label=('Last Name'),
        min_length=1,
        widget=forms.TextInput(attrs={'placeholder': ('Last name')}),
    )
```
I think the only reason to assign fields in `__init__` is if they are conditional or if you need to process arguments somehow.

Reply:
Cool, changed in [b16d311](https://github.com/OpenImaging/miqa/pull/422/commits/b16d311d9c04f339896d94f7087b55869e87c201)